tests/baselines/reference/checkJsxChildrenCanBeTupleType.errors.txt

```diff
@@ -1,13 +1,8 @@
-tests/cases/conformance/jsx/checkJsxChildrenCanBeTupleType.tsx(17,18): error TS2769: No overload matches this call.

```

Comment:
I still need to fix this error.

Reply:
I did a lot of further testing and it seems like the reason why the previous errors disappear, is that in `checkApplicableSignatureForJsxOpeningLikeElement` we call `checkTypeRelatedTo` without an `errorNode` so this error is not raised.
If we pass the `errorNode` down, however, other errors change. I'm not sure what behavior we want here.